modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/dtx/DistributedTransaction.java

```diff
@@ -253,7 +260,11 @@ public void enqueueMessage(AndesMessage andesMessage, AndesChannel andesChannel)
                 }
             }
         } else {
-            QpidAndesBridge.messageReceived(andesMessage, andesChannel);
+            try {
+                QpidAndesBridge.messageReceived(andesMessage, andesChannel);
+            } catch (AndesException e) {
+                LOGGER.error("Non-transactional message publishing failed", e);
```

Comment:
Is it correct to say "Non-transactional message publishing failed"?

Reply:
Yes. If we do a message publishing without sending a dtx.start, it is non-transational.